api/src/main/java/io/minio/MinioProperties.java

```diff
@@ -26,7 +26,7 @@
 import java.util.logging.Logger;
 
 /** Identifies and stores version information of minio-java package at run time. */
-enum MinioProperties {
+public enum MinioProperties {
```

Comment:
Is it required to be `public`?

Reply:
yes for adding USER_AGENT inside MinioAdminClient